src/libtiled/mapreader.h

```diff
@@ -95,6 +97,9 @@ class TILEDSHARED_EXPORT MapReader
      */
     QString errorString() const;
 
+	int GetNumTilesetSourceFiles();
+	QString* GetTilesetSourceFiles();
```

Comment:
Also, according to qt and Tiled styles, these funcs should start with lowercase and without 'get' prefix.